src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -78,4 +80,56 @@ public void testGroupByInQuery() {
     private JSONObject executeJdbcRequest(String query){
         return new JSONObject(executeQuery(query, "jdbc"));
     }
+
+    @Test
+    public void numberOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT ABS(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT abs(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age IS NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void trigFunctionNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT Cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc"),
+                executeQuery("SELECT cos(age) FROM elasticsearch-sql_test_index_account/account " +
+                                "WHERE age is NOT NULL LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void stringOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery(
+                        "SELECT SubStrinG(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc"),
+                executeQuery(
+                        "SELECT substring(lastname, 0, 2) FROM elasticsearch-sql_test_index_account/account " +
+                                "ORDER BY age LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void binaryOperatorNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT lastname FROM elasticsearch-sql_test_index_account/account " +
+                        "WHERE (age IS NOT NULL) AND (gender = 'M') ORDER BY age LIMIT 5", "jdbc"),
+                executeQuery("SELECT lastname FROM elasticsearch-sql_test_index_account/account " +
+                        "WHERE (age IS NOT NULL) and (gender = 'M') ORDER by age LIMIT 5", "jdbc")
+        );
+    }
+
+    @Test
+    public void dateFunctionNameCaseInsensitiveTest() {
+        assertEquals(
+                executeQuery("SELECT insert_time FROM elasticsearch-sql_test_index_online/online " +
+                        "WHERE date_FORMAT(insert_time, 'yyyy-MM-dd', 'UTC') > '2014-01-01'", "jdbc"),
```

Comment:
In your issue #233, I noticed that Andy also mentioned https://github.com/opendistro-for-elasticsearch/sql/issues/176. Could you change this test case by using `date_format` with different cases in `GROUP BY` and `ORDER BY` too? So we can make sure we support case insensitive function name present anywhere. Reference: https://github.com/opendistro-for-elasticsearch/sql/blob/2ef99b3e0a822aea498525a39992ec512eb25636/src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/DateFormatTest.java#L99

Reply:
@dai-chen yes sure. will do it, thanks!